src/device_builder.rs

```diff
@@ -106,27 +107,60 @@ impl<'a, B: UsbBus> UsbDeviceBuilder<'a, B> {
         self
     }
 
+    /// Sets **extra** Language ID for device.
+    ///
+    /// Since "en_US"(0x0409) is implicitly embedded, you just need to fill other LangIDs
+    ///
+    /// Default: (none)
+    pub fn extra_lang_ids(mut self, extra_lang_ids: &'a [u16]) -> Self {
```

Comment:
Also, it would be incredibly useful if we just enumerated the `LangId` as an enum that was exposed to the user, so they didn't have to go explicitly look up all the lang-id values

Reply:
ok, I will try dump the LANGID list from [MS-LCID] into the enum after I modify other parts.

[MS-LCID]: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f

Reply:
If it starts looking like a big task, don't worry about it and spawn an issue to make it for later :)

Reply:
got it, thanks.